javascript/example/DRACOLoader.js

```diff
@@ -15,6 +15,7 @@
 'use strict';
 
 THREE.DRACOLoader = function(manager) {
+    this.decoder = THREE.DRACOLoader.getDecoder();
```

Comment:
Thanks, I'll get it updated in the three.js repo as well. 

I agree that only maintaining THREE.DRACOLoader there would be probably better; do you have specific tasks ought to happen first? E.g. does the API need to be more finalized first? Can start another issue to discuss, too.